.circleci/config.yml

```diff
@@ -34,7 +34,7 @@ jobs:
       - run:
           name: Postinstall
           command: |
-            yarn setup
+            yarn
```

Comment:
`yarn setup` was a previous script that we used to include lavamoats allow-scripts but it seems it's uneeded and most projects are moving away from using it 
```
"setup": "yarn install && yarn allow-scripts",
```